phpstan.dev.neon

```diff
@@ -3,3 +3,6 @@ parameters:
     paths:
       - lib/
       - pages/
+
+    ignoreErrors:
+      - '#.*getValueType\(\) expects class\-string, string given\.$#'
```

Comment:
will be fixed on next phpstan major version: https://github.com/phpstan/phpstan-src/pull/1754